pkg/server/bundle/client/updater.go

```diff
@@ -44,11 +48,11 @@ func NewBundleUpdater(config BundleUpdaterConfig) BundleUpdater {
 	}
 }
 
-func (u *bundleUpdater) UpdateBundle(ctx context.Context) (time.Duration, error) {
+func (u *bundleUpdater) UpdateBundle(ctx context.Context) (*bundleutil.Bundle, *bundleutil.Bundle, error) {
 	localBundle, err := fetchBundle(ctx, u.c.DataStore, u.c.TrustDomain)
 	if err != nil {
 		u.c.Log.WithError(err).Error("Failed to fetch local bundle")
```

Comment:
I know this is old code, but perhaps we could just return the errors in this function and log them in `runUpdater()`. 
What do you think?

Reply:
That's a good suggestion. I'm happy to make that change.